app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/fragments/mvp/FragmentDialogModel.kt

```diff
@@ -1,5 +1,6 @@
 package com.puzzlebench.clean_marvel_kotlin.presentation.fragments.mvp
 
+import com.puzzlebench.clean_marvel_kotlin.presentation.fragments.mvp.contracts.FragmentDialogContracts
```

Comment:
are you using this import? the only change here is this adding

Reply:
yes, i'm using contracts for the model of FragmentDialogCharacter, is the only model that has a contract.